src/compas_timber/planning/sequencer.py

```diff
@@ -268,7 +269,7 @@ class SimpleSequenceGenerator(object):
 
     Parameters
     ----------
-    assembly : :class:`compas_timber.assembly.TimberAssembly`
+    assembly : :class:`compas_timber.assembly.TimberAssembly` or :class:'compas.datastructures.Assembly'
```

Comment:
nitpicky but technically `TimberAssembly` IS an `Assembly` so you could just leave `Assembly` here